addons/crm_livechat/static/tests/mock_server/mock_models/res_groups.js

```diff
@@ -0,0 +1,11 @@
+import { serverState, webModels } from "@web/../tests/web_test_helpers";
+
+export class ResGroups extends webModels.ResGroups {
+    _records = [
+        ...this._records,
+        {
+            id: serverState.groupSalesTeamId,
+            name: "Sales Team",
+        },
+    ];
+}
```

Comment:
I don't like global data available for all tests, this make the test too magic because it relies on implicit data.

It's fine for overly used things like `serverState.partnerId` as almost all tests need self authenticated, but here I much prefer the test explicitly creates the res group.